x/bank/keeper/migrations.go

```diff
@@ -31,5 +32,7 @@ func (m Migrator) Migrate2to3(ctx sdk.Context) error {
 
 // Migrate3to4 migrates x/bank storage from version 3 to 4.
 func (m Migrator) Migrate3to4(ctx sdk.Context) error {
```

Comment:
Can we add an helper, that gets already migrated bank params (so current v047 users), get the send enabled params from x/params and update the bank params?
This way affected users (I've checked, found none, but just in case), can use it.

Reply:
Not sure if you mean add a public helper in app module, or do you have any example I should put it in.

Reply:
This is almost what I meant, yes, but no need to have anything in AppModule.

You can keep `Migrate3to4` as you had it, with the fix, and move `MigrateSubspaceParams` here (package migration), named `MigrateSendEnabledParams` with a comment explaining this is only needed for chains having migrated from <= v0.47 to v0.47.0-5